test/test_deployment_utils.py

```diff
@@ -1452,6 +1457,71 @@ def test_get_deployment_config_ff_hotseat_orchestrated():
                                            f' Processing mode: default')
 
 
+@using_fresh_ff_state()
+@mock.patch.object(deployment_utils_module, "compute_ff_prd_env")
+@mock.patch.object(deployment_utils_module, "compute_cgap_prd_env")
+def test_actual_env_settings(mock_compute_cgap_prd_env, mock_compute_ff_prd_env):
+    def dont_compute_cagp(*args, **kwargs):
+        ignored(args, kwargs)
+        raise AssertionError("Should not be trying to call compute_cgap_prd_env")
+    mock_compute_cgap_prd_env.side_effect = dont_compute_cagp
+    mock_compute_ff_prd_env.return_value = 'fourfront-prd'
+
+    assert EnvUtils.PRD_ENV_NAME == 'fourfront-prd'
+    assert EnvUtils.STG_ENV_NAME == 'fourfront-stg'
+
+    def describe(config):
+        return ('SKIP'
+                if config['SKIP'] else
+                ",".join((['WIPE_ES'] if config['WIPE_ES'] else []) +
+                         (['STRICT'] if config['STRICT'] else [])))
+
+    class MyCommandArgs(MockedCommandArgs):
+        VALID_ARGS = ['skip', 'wipe_es', 'strict']
+
+    def compute_and_show(kind, env, wipe_es=False):
+        info = "-------------------------"
+        try:
+            config = _get_deploy_config(env=env, args=MyCommandArgs(wipe_es=wipe_es))
+            info = describe(config).ljust(14)
+        finally:  # show even if we err
+            print(f"{info} env={env.ljust(20)} wipe_es={'Y' if wipe_es else 'N'}"
+                  f" full={full_env_name(env).ljust(20)} kind={kind}")
+        return config
+
+    print()  # Start on a fresh line
+    for wipe_es in [False, True]:
+        print(f"wipe_es={wipe_es}")
+        for kind_and_envs in [['production', ['fourfront-prd', 'prd', 'data']],
+                              ['staging', ['fourfront-stg', 'stg', 'staging']],
```

Comment:
Seems like `fourfront-production-green` and `fourfront-production-blue` should be here?